src/main/java/at/hannibal2/skyhanni/features/inventory/ItemDisplayOverlayFeatures.kt

```diff
@@ -148,7 +148,7 @@ class ItemDisplayOverlayFeatures {
         }
 
         if (SkyHanniMod.feature.inventory.itemNumberAsStackSize.contains(12)) {
-            if (item.getInternalName_old() == "LARVA_HOOK") {
+            if (itemName.contains("Larva Hook")) {
```

Comment:
I would prefer to use the internal name for checks like this in the future (I know internal name check is slower than item name check, but hypixel tends to rename the item names sometimes and internal names dont change) also this whole method should get an cache so this doenst call each frame